src/containers/board/Board.js

```diff
@@ -4,15 +4,10 @@ import Spell_field from'../spell_field/Spell_field.js'
 import Monster_field from'../monster_field/Monster_field.js'
```

Comment:
Use camelcase for JavaScript / React. 

i.e. MonsterField (for component name, file name, and directory name)